src/JsonSchema/Constraints/ConstraintInterface.php

```diff
@@ -59,5 +59,10 @@ public function isValid();
      * @param mixed            $i
      * @throws \JsonSchema\Exception\ExceptionInterface
      */
+    public function _check(&$value, $schema = null, JsonPointer $path = null, $i = null);
```

Comment:
> My position on this kind of thing is that breaking a five-year-old public API immediately and without warning when a couple of aliases can allow for graceful deprecation over the next few releases is unconscionable, regardless of how trivial the fix on their end. Users deserve more respect than that.

You're new to this project. The main API has gone through a lot of changes. It is not 5 years old. The API as you know it now is about three months old. We had a breaking change just a few months ago that was more dramatic than this (they changed the API such that instead of feeding storage and uriretriever instances and so on to Validator, you pass them to Factory). To be honest that one went in a little faster than I would have liked--the checkmode got swept along as well for no real reason, which led us to our current situation. But we bumped the major version, and had no complaints.